packages/block-editor/src/components/colors-gradients/panel-color-gradient-settings.js

```diff
@@ -140,10 +164,7 @@ const PanelColorGradientSettings = ( props ) => {
 	) {
 		return <PanelColorGradientSettingsInner { ...props } />;
 	}
-	if ( props.__experimentalHasMultipleOrigins ) {
```

Comment:
I don't understand the changes on this component, can you detail a bit more?

For me you had two options here:

 - Either you could have forced all the controls to be "multiple" or "single" (you choose here the shape of the colors/gradients prop to provide to the lower level components from the components package)
 - Or keep the `__experimentalHasMultipleOrigins` prop at this level (block-editor package components) and construct the shape of the palettes depending on this prop, leaving the choice to the consumer of these components.